src/components/Card/Card.js

```diff
@@ -1,16 +1,15 @@
 import React from 'react';
 import './card.css'
 
-const Card = (movie, showSelectedMovie, convertDate) => {
+const Card = (movie, showSelectedMovie) => {
```

Comment:
So remember - if you are going to destructure the props, you have to wrap them in curly braces. Otherwise, your component will assume that you are naming the props object "movie".